hasher-matcher-actioner/mypy.ini

```diff
@@ -1,26 +1,6 @@
-[mypy]
-
 [mypy-threatexchange.*]
```

Comment:
I added this to suppress the CI issues but it feels too general for the long term which is why I kept the  `[mypy-threatexchange.hashing.*]` etc because I was hoping we'd be able to remove the root ignore it eventually...